pkg/genericapiserver/server/openapi/openapi.go

```diff
@@ -33,13 +33,15 @@ import (
 
 const (
 	OpenAPIVersion = "2.0"
+	ExtensionTags  = "x-kubernetes-tags"
```

Comment:
I would not use acronyms in the extension name - should be "group-version-kind" or "kind"